### PR TITLE
fix(header): added bigger logo with animation

### PR DESCRIPTION
**Pull request by gfellerph**
As per request from branding, the logo should be the topmost element on the page. To handle the disappearing meta-navigation, the logo animation was introduced.

### CODE REVIEW

File: packages/internet-header/src/components/post-internet-header/post-internet-header.tsx [hunk 1]
```diff
@@ -436,6 +441,7 @@ export class PostInternetHeader {
               orientation="horizontal"
               class="hidden-lg"
               full-width={this.fullWidth}
+              ref={el => (this.metaNav = el)}
```
Comment (alizedebray) [inline packages/internet-header/src/components/post-internet-header/post-internet-header.tsx:444]: Good to know!

**Review by alizedebray — APPROVED** [review]

File: packages/internet-header/src/components/post-internet-header/logo-animation/logo-animation.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  @include mixins.max(lg) {
+    --logo-scale: 1 !important;
+  }
+}
+
+post-logo {
+  height: var(--header-height);
+  transform-origin: bottom left;
+  transform: scale(var(--logo-scale));
```
[the remark this answers is not in the document]
Comment (gfellerph) [reply packages/internet-header/src/components/post-internet-header/logo-animation/logo-animation.scss:19]: I tested with a few values
- 10ms: no real difference to no transition
- 50ms: lags a bit behind but does not help to make it smooth
- 100ms: feels unresponsive

I'd go without for the moment

File: packages/internet-header/src/components/post-internet-header/logo-animation/logo-animation.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      scale = Math.max(
+        (adjustedHeaderHeight - Math.max(window.scrollY, 0)) /
+          (adjustedHeaderHeight - target.clientHeight),
+        1,
+      );
+    }
+    headerRef.style.setProperty('--logo-scale', scale.toString());
+  };
+
+  const debounced = debounce(150, handleScroll);
```
Comment (oliverschuerch) [inline packages/internet-header/src/components/post-internet-header/logo-animation/logo-animation.ts:26]: Would it make the transition smoother, if you reduce/remove the debounce?

File: packages/internet-header/src/components/post-meta-navigation/post-meta-navigation.scss [hunk 4]
```diff
@@ -40,11 +41,22 @@
       --separator-display: block;
       --separator-height: #{functions.px-to-rem(34px)};
     }
+
+    &::before {
+      content: '';
```
Comment (oliverschuerch) [inline packages/internet-header/src/components/post-meta-navigation/post-meta-navigation.scss:46]:
```suggestion
      display: block;
      content: '';
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]
When having the header in the full-sticky mode, with meta navigation enabled and scrolling through the page, the logo animation is executed as well.
This needs to be fixed, because the animation is not necessary then.

File: packages/internet-header/src/components/post-internet-header/logo-animation/logo-animation.ts (hunk 3 above)
Comment (gfellerph) [reply packages/internet-header/src/components/post-internet-header/logo-animation/logo-animation.ts:26]: For resizing the window, animation smootheness is not important.

**Comment by oliverschuerch** [comment]
@gfellerph The logo works correctly, but the animation behaviour is not updated correctly within the documentation, when a user switches the stickyness.
Maybe it could help, to trigger a window resize event or something, whenever the story is rerendered.

**Comment by gfellerph** [comment]
> @gfellerph The logo works correctly, but the animation behaviour is not updated correctly within the documentation, when a user switches the stickyness. Maybe it could help, to trigger a window resize event or something, whenever the story is rerendered.

done @oliverschuerch

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**